eventually/team/models.py

```diff
@@ -0,0 +1,125 @@
+"""
+Team model
+=======================
+
+This module implements class that represents the team entity.
+"""
+from django.db import models
+from django.db import IntegrityError
+
+
+class Team(models.Model):
+
+    """
+    .. class:: Team
+    Describing of team entity.
+    Attributes:
+    ===========
+
+        :param name: Name of the certain team.
+        :type name: string
+        :param description: Describing core information of the team.
+        :type description: string
+        :param image: Describing link to image of the team.
+        :type description: string
+        :param create_date: The date when the
+                            certain team was created.
+        :type: create_date: datetime
+        :param update_date: The date when
+                    the certain team was last time edited.
+    """
+    name = models.CharField(max_length=30)
+    description = models.TextField(blank=True)
+    image = models.CharField(max_length=300, blank=True)
+    created_at = models.DateTimeField(auto_now_add=True, editable=False)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    def to_dict(self):
+        """
+        Method that converts team object to dictionary.
+        :return: dictionaty with team's information
+        """
+
+        return {
+            'name': self.name,
+            'description': self.description,
+            'image': self.image,
+            'created_at': self.created_at,
+            'updated_at': self.updated_at,
+        }
+
+    @staticmethod
+    def get_by_id(team_id):
+        """
+        Static method that returns team objects according to the accepted id.
+        Return comment, found by id.
+        :param team_id: team's id
+        :type team_id: int
+        :return: Team object instance or None if object does not exist
+        """
+
+        try:
+            team = Team.objects.get(id=team_id)
+            return team
+        except Team.DoesNotExist:
+            return None
+
+    @staticmethod
+    def create(name=None, description='', image=''):
+        """
+        Static method that creates instance of Team class and creates databes
+        row with the accepted info.
+        :param name: team's name
+        :type name: str
+        :param description: team's description
+        :type description: str
+        :param image: team's image
+        :type image: str
+        :return: Team object instance or None if object does not exist
+        """
+
+        team = Team()
+        team.name = name
+        team.description = description
+        team.image = image
+        try:
+            team.save()
+            return team
+        except IntegrityError:
```

Comment:
Why is IntegrityError, and not ValueError?

Reply:
Done